mlx_lm/models/llama.py

```diff
@@ -24,6 +24,7 @@ class ModelArgs(BaseModelArgs):
     num_key_value_heads: Optional[int] = None
     attention_bias: bool = False
     mlp_bias: bool = False
+    quantization_config: Optional[Dict[str, Union[str, int]]] = None
```

Comment:
Could you remove that?

Reply:
This is important for the tests.

I can remove it from the config and hardcoded it for the tests if you would like.